packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -164,6 +166,20 @@ export function buildRootObject() {
       return zcfMint;
     };
 
+    function updateNotiferFrom(updater, notifier, nextCount = undefined) {
+      notifier.getUpdateSince(nextCount).then(
```

Comment:
Isn't this `notifier` actually a reference to a remote notifier, the one in the zoe vat? If so, this would need to be `E(notifier).`

Reply:
moot.